image/preseed/preseed_uc20_test.go

```diff
@@ -354,7 +419,14 @@ func (s *preseedSuite) TestRunPreseedUC20ExecFormatError(c *C) {
 	err := ioutil.WriteFile(mockChrootCmd.Exe(), []byte("invalid-exe"), 0755)
 	c.Check(err, IsNil)
 
-	opts := &preseed.PreseedOpts{PreseedChrootDir: tmpdir}
-	err = preseed.RunUC20PreseedMode(opts)
+	opts := &preseed.CoreOptions{
+		PrepareImageDir: tmpdir,
+	}
+	popts := &preseed.PreseedCoreOptions{
+		CoreOptions:      *opts,
+		PreseedChrootDir: tmpdir,
+	}
```

Comment:
nitpick: there are a couple of places in the tests where we do this, but it feels a bit weird to declare the inner struct as a pointer if we are copying it inside the outer struct anyway.
```suggestion
	popts := &preseed.PreseedCoreOptions{
		CoreOptions:      preseed.CoreOptions{
			PrepareImageDir: tmpdir,
		},
		PreseedChrootDir: tmpdir,
	}
```

Reply:
nice, much cleaner now.
Thank you @mardy

Reply:
No problem at all, fixed now :)